.github/workflows/labeler.yml

```diff
@@ -18,13 +20,20 @@ jobs:
 
       - uses: actions/checkout@v3
 
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Install dependencies
+        run: npm ci
+
       - name: Build 
         run: npm run build
 
       - uses: ./
         name: Label issue or pull request
         with:
           label_name: community
-          label_color: 5319E7
+          label_color: '5319e7'
```

Comment:
Is it necessary to have this string in quotes?

Reply:
In this case yeah! So this is interesting because the hex we are using gets interpreted as an exponential if not wrapped.